src/components/Animation/Animation.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+
+export default class Animation extends React.Component {
```

Comment:
Is this component still needed?

Reply:
Great question. I couldn't get the example to load up in styleguidist unless there was an actual component. I thought that odd. I am curious about different ways of abstracting some of the inner working of `react-move` for reuse and easy of implementation, but I didn't want to get caught up in that for this.